operations/helm/charts/mimir-distributed/templates/memcached/_memcached-statefulset.tpl

```diff
@@ -84,8 +84,7 @@ spec:
               name: client
           args:
             - -m {{ .allocatedMemory }}
-            - -o
-            - modern
+            - --extended=modern,track_sizes
```

Comment:
`modern` isn't actually needed since the settings it changes are already the default in the version of Memcached we use.

Reply:
If it's been there since a couple of helm releases (6 months-year), I suppose we can just remove it.

Reply:
I'm thinking about some org that might have had to keep an earlier version of the memcached image because of some internal approvals process for example. But if it's been the default for a long time, the chance of them using a year-old memcached version is lower.

Reply:
Yeah, it's not a big deal either way. Just thought I'd point it out. I'll leave it for now.